port/ztpf/omrosbacktrace_impl.c

```diff
@@ -246,6 +252,7 @@ omrintrospect_backtrace_symbols_raw(struct OMRPortLibrary *portLibrary,
 		uintptr_t symbol_offset = 0;
 		uintptr_t module_offset = 0;
 		const char *symbol_name = "";
+		char asciiSymbolBuffer[512], asciiModuleBuffer[512];
```

Comment:
Can you declare each variable on it's own line?

Reply:
Done.